src/control/CMakeLists.txt

```diff
@@ -42,6 +42,20 @@ install(DIRECTORY include/
   DESTINATION include/
 )
 
+if(BUILD_TESTING)
```

Comment:
Any way to include this section without duplicating in each of the CMakeLists.txt files? Perhaps include the boilerplate stuff that's not specific to a module.

Reply:
Yeah. I agree. I've already got a few things I want to improve with the test invocation. I'll try to think of a way to do this. It might require having a seperate package with CMake utilities that every other package imports.